package.json

```diff
@@ -124,8 +127,10 @@
     "aurelia-metadata": "^1.0.4",
     "aurelia-pal": "^1.3.0",
     "aurelia-templating": "^1.8.2",
-    "aurelia-templating-resources": "^1.7.1",
-    "i18next": "^14.1.1"
+    "aurelia-templating-resources": "^1.7.1"
+  },
+  "peerDependencies": {
+    "i18next": ">=20.3.2"
```

Comment:
We'll gonna need to add this as devDependency as well otherwise you can't run npm test on a fresh new instance

Reply:
Done.